### ISSUE TITLE
TODO Fix badge links in readme

### ISSUE THREAD

**Issue by tablatronix** [tzapu/WiFiManager#1376]
Badges should link to pages

**Comment by tablatronix** [comment]
![C7ABC491-52D5-4545-86E7-6A1B3FECBA26](https://user-images.githubusercontent.com/807787/174159716-986cc41d-105a-4d52-8109-4d62d991979a.png)

link to the images and not actual pages

**Comment by JoergAJ** [comment]
Links to pages at espressif  like https://www.espressif.com/en/products/socs/esp32-c3 for the esp32c3 ?

**Comment by tablatronix** [comment]
Yeah but mostly the CI test status pages, but it would be cool if they are linked to some reference maybe?

**Comment by seanhagen** [comment]
@tablatronix like this: https://github.com/seanhagen/WiFiManager/tree/1376-fix-badge-links-in-readme ? I put all the badges on one line, added a link to the releases page for the "Release" badge, and put the chat & discord badges on the same line. All the other badges already have links that *seem* to go to the right places, but I'm not 100% sure

**Comment by tablatronix** [comment]
Looks bad on mobile

**Comment by aumpatel2624** [comment]
hey I want to contribute in solving this issue.

**Comment by tablatronix** [comment]
Do it

**Comment by tablatronix** [comment]
And feel free to add anything else to the readme or wiki it needs all the new updated stuff

**Comment by belanasaikiran** [comment]
Is this fixed or needs to be fixed ? What can I help with ?

**Comment by aumpatel2624** [comment]
It still needs to be fixed I guess

**Comment by Ujjwal2882** [comment]
is that issue is solved
